src/styles/blocks/header.scss

```diff
@@ -0,0 +1,64 @@
+.header {
+  display: flex;
+  background: #fff;
+  justify-content: space-between;
+  align-items: center;
+  height: 60px;
+  padding: 0 50px;
+  box-shadow: 2px 2px 4px rgba(0, 0, 0, 0.1);
+
+  &__logo {
+    width: 40px;
+    height: 40px;
+  }
+
+  &__link {
+    display: flex;
+  }
+
+  &-nav {
+    display: flex;
+    height: 60px;
+    align-items: center;
+
+    &__list {
+      display: flex;
+      list-style: none;
+      font-family: Roboto, sans-serif;
+      font-size: 12px;
+      font-weight: 500;
+      text-transform: uppercase;
+      gap: 20px;
+      padding-inline-start: 0;
+    }
+
+    &__link {
+      display: flex;
+      text-decoration: none;
+      color: $main-text-color;
```

Comment:
The color variable '$main-text-color' should be defined in the 'variables.scss' file to maintain consistency and to make it easier to update or change the color scheme of the application.